platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/HtmlToSpansParser.kt

```diff
@@ -145,6 +158,68 @@ internal class HtmlToSpansParser(
                 text.setSpan(span, start, text.length, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
                 text.removeSpan(last)
             }
+            "pre" -> {
+                val last = getLast<CodeBlock>() ?: return
+                var start = text.getSpanStart(last)
+                text.removeSpan(last)
+
+                if (start > 0) {
+                    if (text[start] == ZWSP) {
+                        text.replace(start, start + 1, "\n")
+                    } else {
+                        text.insert(start, "\n")
+                    }
+                    start++
+                }
```

Comment:
I guess this could be extracted to a function if it's used for all block nodes?

Reply:
b1541c7